### DISCUSSION TITLE
Auto Update

### DISCUSSION THREAD

**Discussion by escfoe2** [supermerill/SuperSlicer#1823] [Community contribution]
Just a friendly note to consider using Squirrel.Windows to automatically push updates.  It installs the application at the user level so no UAC is necessary.  I've used it personally so I know it works and it works well.

https://github.com/Squirrel/Squirrel.Windows

**Comment by supermerill** [comment]
Seems a bit intrusive and restrictive.
I prefer to not add windows-only code inside the app as much as possible.
And it seems to mess with the compiling things, so a bit painful to integrate.

**Reply by escfoe2** [reply]
I'm willing to bet over 90% of the users are windows users.  You'd be serving the largest portion of the community up front with an auto updater.  It'll simply make the application more user friendly.   You don't have to go with Squirrel.  Having some sort of built in updater would be preferred though, however you choose to do it.

**Comment by tloesch** [comment]
@escfoe2 I created an updater for windows  because i also don't like to update it manually.

At the moment I don't know how stable it is because it got created for personal use. 

I released it now on github. 
Feel free to try it out. 

https://github.com/tloesch/superslicer-updater

**Reply by supermerill** [reply]
Thanks, i'll try.
Currently, i'm trying to sort out the signing certificates, so the UAC won't trigger.
